UOP1_Project/Assets/Scripts/UI/FadeManager.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class FadeManager : MonoBehaviour
+{
+	[Header("Fadeout Event")]
+	[SerializeField] private FadeChannelSO _fadeChannelSO = default;
+
+	[Header("Fadeout Canvas")]
```

Comment:
Wouldn't just `Fade` be a more appropriate name?
```suggestion
	[Header("Fade Event")]
	[SerializeField] private FadeChannelSO _fadeChannelSO;

	[Header("Fade Canvas")]
```
also change the `[Header("Fade Canvas")]` to `[Header("Fade Image")]` if you apply my other suggestion

Reply:
I removed `= default;` because my IDE (Jetbrains Rider) told me this was redundant
A variable will always be set to the default value so `= default` only makes sense if you want to give a function-parameter a default value

Reply:
> 
> 
> Wouldn't just `Fade` be a more appropriate name?
> 
> also change the `[Header("Fade Canvas")]` to `[Header("Fade Image")]` if you apply my other suggestion

Will do.